ipatests/test_integration/test_membermanager.py

```diff
@@ -212,3 +212,54 @@ def test_hostgroup_member_manager_nopermission(self):
             "'write' privilege to the 'memberManager' attribute of entry"
         )
         assert expected in result.stdout_text
+
+    def test_member_manager_upgrade_scenario(self):
+        """tmp"""
+        user = "idmuser"
+        password = "Secret123"
+        group1 = "role-groupmanager"
+        group2 = "role-usergroup-A"
+
+        master = self.master
+
+        tasks.kinit_admin(master)
+        tasks.group_add(master, group1)
+        tasks.group_add(master, group2)
+        tasks.create_active_user(master, user, password)
+
+        tasks.kinit_admin(master)
+        master.run_command(["ipa", "group-add-member", group1, "--user", user])
+        master.run_command(["ipa", "group-add-member-manager", "--groups",
+                            group1, group2])
+
+        replica = self.replicas[0]
+        base_dn = "dc=%s" % (",dc=".join(replica.domain.name.split(".")))
+        ldif_entry = tasks.textwrap.dedent("""dn: cn=hostgroups,cn=accounts,
+        {base_dn}
+        changetype: modify
+        delete: aci
+        aci: (targetattr = "member")(targetfilter =
+        "(objectclass=ipaHostGroup)")(version 3.0; acl
+        "Allow member managers to modify members of host groups"; allow (write)
+        userattr = "memberManager#USERDN" or userattr =
+        "memberManager#GROUPDN";)
+
+        dn: cn=groups,cn=accounts,{base_dn}
+        changetype: modify
+        delete: aci
+        aci: (targetattr = "member")(targetfilter =
+        "(objectclass=ipaUserGroup)")(version 3.0; acl
+        "Allow member managers to modify members of user groups"; allow (write)
+        userattr = "memberManager#USERDN" or userattr =
+        "memberManager#GROUPDN";)""").format(base_dn=base_dn)
+        tasks.ldapmodify_dm(master, ldif_entry)
+
+        tasks.kinit_as_user(master, user, password)
+        # should fail
+        with tasks.pytest.raises(tasks.subprocess.CalledProcessError):
+            master.run_command(["ipa", "group-add-member" "--users",
+                                "admin", group2])
```

Comment:
There is a missing comma between the arguments in line 260:
```
master.run_command(["ipa", "group-add-member" "--users",
```
and the error should not be a CalledProcessError. You can add raiseonerr=False to the call to run_command and check the return code and stderr_text instead.